src/glfw-sandbox/drawings.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
```

Comment:
General tip: In C++, I think Nico called the header files .hpp - to differentiate from .h digestible by C.

General tip: .hpp file should contain just the interface, accompanying .cpp should contain the implementation then. Do you remember how we achieved that?

Reply:
I was generally also confused with this, because I saw numerous times .h files accompanied with .cpp
I believe you just use something like a class definition in .hpp file
e.g my_awesome_class.hpp
```
{
    class my_awesome_class
    {
    public:
        void do_something();
    };

}
```
then in implement_my_awesome_class.cpp

```
#include "my_awesome_class.hpp" 
#include <iostream> 

void my_awesome_class::do_something()
{
    cout << "Yes I am doing something!" << endl;
}
```

Again I am not sure :)

Reply:
Exactly!!!

I discussed this with Hanka at work and she mentions that this allows to keep the hpp files free of unnecessary includes, among other things - like, you can quickly overview the interface.